### PR TITLE
Minor LaTeX Table Fixes

### PR DESCRIPTION
**Pull request by samm82**
- Bolded all non-chunk table headers in LaTeX
   - This also bolds the headers of the trace tables; is this OK/desired? @JacquesCarette @smiths
- Increased chunk column width so everything fits on one line

_Closes #1640 and closes #1638_

### CODE REVIEW

**Comment by smiths** [comment]
The bold tables look good to me, as does the output constraints.  (I can see that we could use the white space more effectively with a narrower column and a new line between "Output" and "Constraints", but  I think we don't want to get too caught up in fine tuning the appearance of the generated documentation.  The changes in this PR already make the LaTeX docs look better.)

**Comment by samm82** [comment]
@smiths Unfortunately, line splitting in LaTeX isn't as simple as I'd hoped it would be, or else I would have gone that route.

**Comment by smiths** [comment]
There are ways to tell LaTeX how to format the contents of a cell.  It seems to me that I've used something called ragged right for this in the past.  Ragged right is discussed at:

https://tex.stackexchange.com/questions/12703/how-to-create-fixed-width-table-columns-with-text-raggedright-centered-raggedlef

Among other places.

It might let us reduce the column width, but still have it look nice.  If a quick check of this doesn't work @samm82, I'm fine with not spending any more cycles on this.  The LaTeX tables already look better than they did.

**Comment by samm82** [comment]
`raggedright` is actually a lot simpler than I thought, so I went that route now instead. I might have spent a little too long making the tables the "perfect" size, but at least no one should have to fix it in the future.

File: code/drasil-printers/Language/Drasil/TeX/Print.hs [hunk 1]
```diff
@@ -197,7 +197,7 @@ makeTable :: [[Spec]] -> D -> Bool -> D -> D
 makeTable lls r bool t =
   pure (text ("\\begin{" ++ ltab ++ "}" ++ (brace . unwords . anyBig) lls))
   %% pure (text "\\toprule")
-  %% makeRows [head lls]
+  %% makeRows [map (\x -> S "\\textbf{" :+: x :+: S "}") (head lls)]
```
Comment (JacquesCarette) [inline code/drasil-printers/Language/Drasil/TeX/Print.hs]: You really shouldn't hand-build latex like this, but rather use the facilities in the pretty-printer built for that purpose.

File: code/drasil-printers/Language/Drasil/TeX/Print.hs [hunk 2]
```diff
@@ -333,7 +333,7 @@ makeDefTable :: PrintingInformation -> [(String,[LayoutObj])] -> D -> D
 makeDefTable _ [] _ = error "Trying to make empty Data Defn"
 makeDefTable sm ps l = vcat [
   pure $ text 
-  $ "\\begin{tabular}{p{"++show colAwidth++"\\textwidth} p{"++show colBwidth++"\\textwidth}}",
+  $ "\\begin{tabular}{>{\\raggedright}p{"++show colAwidth++"\\textwidth}>{\\raggedright\\arraybackslash}p{"++show colBwidth++"\\textwidth}}",
```
Comment (JacquesCarette) [inline code/drasil-printers/Language/Drasil/TeX/Print.hs]: hard-coded widths like this seem like a bad idea? This doesn't feel future-proof.

Comment (JacquesCarette) [reply code/drasil-printers/Language/Drasil/TeX/Print.hs]: Oh, I see that these are percentages of \textwidth - that's not too bad then.

**Pull request merged**